dbmanager.py

```diff
@@ -31,23 +40,38 @@ def exec_sql(connection, sql):
     :returns None if the connection or sql was None,
              a cursor containing the results of the statement otherwise
     :raises A MySqlError (likely due to an error in the SQL statement syntax)
+    
+    example: 
+        
+        from testdb.py
+        import dbmanager
```

Comment:
@ASAAR Actually I meant 'From the python file testdb.py, do this stuff:' import dbmanager...